internal/config/mount_config.go

```diff
@@ -90,8 +91,9 @@ type EnableHNS bool
 type CacheDir string
 
 type FileSystemConfig struct {
-	IgnoreInterrupts      bool `yaml:"ignore-interrupts"`
-	DisableParallelDirops bool `yaml:"disable-parallel-dirops"`
+	IgnoreInterrupts           bool  `yaml:"ignore-interrupts"`
+	DisableParallelDirops      bool  `yaml:"disable-parallel-dirops"`
+	KernelDirCacheTtlInSeconds int64 `yaml:"kernel-dir-cache-ttl-secs"`
```

Comment:
I think we should handle the case where if value less than -1 is passed to this flag then we should return error to user.